packit_service/worker/helpers/build/copr_build.py

```diff
@@ -323,6 +333,45 @@ def is_custom_copr_project_defined(self) -> bool:
             or self.job_project != self.default_project_name
         )
 
+    def is_forge_project_allowed_to_build_in_copr(self) -> bool:
+        """Is this forge project allowed to build in COPR project?
+        Ask to COPR server if the forge project has been granted permissions.
+
+        Returns:
+            bool: True if the forge project is allowed to build in COPR project.
+        """
+        copr_project = self.api.copr_helper.copr_client.project_proxy.get(
+            self.job_owner, self.job_project
+        )
+        allowed_projects = copr_project["packit_forge_projects_allowed"]
+        allowed = self.forge_project in allowed_projects
+        if not allowed:
+            logger.warning(
+                f"git-forge project {self.forge_project} "
+                f"can't use {self.configured_copr_project} Copr project "
+                f"(Only {allowed_projects} are allowed.)"
```

Comment:
Is this public information? Can we safely show that?

Reply:
Good point, I think this is shown just in our internal logs so I think it is safe and helps us debugging. But I may be wrong.

Reply:
but you can also get this via Copr API, it's public